packages/pwa/app/commerce-api/index.test.js

```diff
@@ -24,6 +25,8 @@ import {
     ocapiFaultResponse
 } from './mock-data'
 
+jest.mock('cross-fetch', () => global.fetch)
```

Comment:
nit, this code implicitly depends on the global mock [here](https://github.com/SalesforceCommerceCloud/pwa-kit/pull/382/files#diff-00df691064d141f54facfe81ca2f50b4656150b5dccb8b53edfb2be26eec8503R15), does it work if you do 
```
import fetch from 'jest-fetch-mock'
jest.mock('cross-fetch', () => fetch)
```